Sources/NIO/Selector.swift

```diff
@@ -41,6 +41,71 @@ private extension Optional {
     }
 }
 
+private struct KQueueEventFilterSet: OptionSet, Equatable {
+    typealias RawValue = UInt8
+
+    let rawValue: UInt8
+
+    static let _none = KQueueEventFilterSet(rawValue: 0)
+    static let except = KQueueEventFilterSet(rawValue: 1 << 1)
+    static let read = KQueueEventFilterSet(rawValue: 1 << 2)
+    static let write = KQueueEventFilterSet(rawValue: 1 << 3)
+
+    init(rawValue: RawValue) {
+        self.rawValue = rawValue
+    }
+}
+
+private struct EPollFilterSet: OptionSet, Equatable {
```

Comment:
Minor note, but the casing here is inconsistent with the rest of the module: should be `Epoll`, not `EPoll`.